tests/fixups/TraceFixup/DynamicLinkLibraryFixup.cpp

```diff
@@ -65,12 +121,40 @@ auto LoadLibraryExImpl = psf::detoured_string_function(&::LoadLibraryExA, &::Loa
 template <typename CharT>
 HMODULE __stdcall LoadLibraryExFixup(_In_ const CharT* libFileName, _Reserved_ HANDLE file, _In_ DWORD flags)
 {
+    LARGE_INTEGER TickStart, TickEnd;
+    QueryPerformanceCounter(&TickStart);
     auto entry = LogFunctionEntry();
     auto result = LoadLibraryExImpl(libFileName, file, flags);
+    QueryPerformanceCounter(&TickEnd);
 
     auto functionResult = from_win32_bool(result != NULL);
     if (auto lock = acquire_output_lock(function_type::dynamic_link_library, functionResult))
     {
+        if (output_method == trace_method::eventlog)

```

Comment:
Is it possible to move the fixup logging in the if(output_method) == trace_method::eventlog block to a separate file that all fixups can reference?  That was we don't have duplicate code?

Reply:
Scott had a similar request but we were unable to come up with a workable solution.  The problem is that the code on the following three lines needs to be inline in the method we are in.  By putting those lines in another function, they will locate the wrong calling function.  What is in place is less than ideal, but we did not come up with a better solution.

Reply:
No worries.  Thank you for looking into it.